packages/binarytree/CHANGELOG.md

```diff
@@ -6,7 +6,30 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 10.0.0-dev-rc.1 - 2025-03-24
+## 10.0.0 (EXPERIMENTAL) - 2025-04-29
+
+**Note:** This library is in an **experimental** stage and should not be used in production!
+
+### Overview
+
+This release is part of the `v10` breaking release round making the `EthereumJS` libraries compatible with the [Pectra](https://eips.ethereum.org/EIPS/eip-7600) hardfork going live on Ethereum `mainnet` on May 7 2025. Beside the hardfork update these releases mark a milestone in our release history since they - for the first time ever - bring the full `Ethereum` protocol stack - including the `EVM` - to the browser without any restrictions anymore, coming along with other substantial updates regarding library security and functionality.
```

Comment:
I get anxious about this phrasing "full stack to the browser" language. Devp2p explicitly does not work in the browser. Can we at something like "the full set of Ethereum primitives"?

Disregard if I'm being too pedantic

Reply:
TBH I think really no one will think about devp2p when reading this sentence, I think this is just our own over-knowledged inside view. 🙂 😋